.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+            product: pkg:generic/hello@${{ env.RELEASE_VERSION }}
```

Comment:
is this a placeholder?

Reply:
Good catch. Yes it was, I've updated it.